### DISCUSSION TITLE
How to re-try publish of packages that failed in previous publish attempt

### DISCUSSION THREAD

**Discussion by petermetz** [ghiscoding/lerna-lite#413] [Q&A] [answered]
Hiya,

The title says it all. I was running a publish script and somewhere in the middle one of the packages failed to publish after the first half already had been published.

Then I fixed what I needed to (in my case I needed to explicitly specify the dist tag as `latest`) and then re-ran the publish script but now it is failing on every package that was successfully published before because those are already published and this makes it error out instead of just finishing the ones that are still unpublished.

For now the only workaround I found was to manually change directories to each package and run `npm publish` myself but I assume there's some combination of flags I can pass in to the publish script to get the "publish what hasn't been and ignore failures of the others" type behavior that I want.

Thank you in advance!

**Comment by ghiscoding** [comment]
Personally I'm not too sure, I never encountered these kind of problems (thankfully), but if you look at Lerna list of issues, you'll find this specific [issue](https://github.com/lerna/lerna/issues/455) which seems to be related and offer ways to deal with this problem.

**Reply by ghiscoding** [reply, marked as answer]
Hello, I just wanted to provide more info on the subject since this problem happened to me yesterday while releasing Lerna-Lite new major [v2.0.0](https://github.com/lerna-lite/lerna-lite/releases/tag/v2.0.0) release. I think in my case it was because I tried to publish while using WSL (Windows Sub-System) and it might have loss connectivity or something in the middle of it... 

The great thing is that we now have a fix that was provided in PR #469 and available since v1.15.0 (which was a PR that was created in Lerna that I replicated in Lerna-Lite) and yesterday was the first time that I really had a chance (or should I say, "forced to use") to try it live and.... it works! The process is that if your process failed and only half of the packages got published, which is what happened to me, you can simply recall the same lerna publish command and it will resume with whatever packages weren't yet published and that's it, it worked :)

in summary, this is no longer an issue 🚀

**Reply by petermetz** [reply]
@ghiscoding Thank you so much for coming back to this and updating it! I've marked your new comment as the answer!
